unit_tests/dbp_testing.py

```diff
@@ -282,7 +282,7 @@ def loadData(self, filename):
             # persist_selectable added 1.3 (mapped_table deprecated)
             tbl = insp.persist_selectable\
                   if hasattr(insp, 'persist_selectable') else insp.mapped_table
-            tbl.drop()
+            tbl.drop(bind=self.dbu.engine)
```

Comment:
A similar change needs to be made for all the calls to `drop_all`...I'm seeing three of them in the testing, but you should search to be sure. They all look to only be called if running under postgres, which is why these got triggered.